src/etc/inc/xmlrpc_client.inc

```diff
@@ -0,0 +1,137 @@
+<?php
+/*
+ * xmlrpc_client.php
+ *
+ * part of pfSense (https://www.pfsense.org)
+ * Copyright (c) 2016 Electric Sheep Fencing, LLC
```

Comment:
Can you please just replace this line by the following to reflect recent changes? Then I'm ready to merge
```
 * Copyright (c) 2016 Rubicon Communications, LLC (Netgate)
```